js/ui/viewSelector.js

```diff
@@ -53,16 +53,18 @@ const ViewsDisplayLayout = new Lang.Class({
     Name: 'ViewsDisplayLayout',
     Extends: Clutter.BinLayout,
 
-    _init: function(entry, allViewActor, searchResultsActor) {
+    _init: function(entry, gc, allViewActor, searchResultsActor) {
         this.parent();
 
         this._entry = entry;
+        this._gc = gc;
         this._allViewActor = allViewActor;
         this._searchResultsActor = searchResultsActor;
 
         this._entry.connect('style-changed', Lang.bind(this, this._onStyleChanged));
         this._allViewActor.connect('style-changed', Lang.bind(this, this._onStyleChanged));
 
+        this._heightAboveGC = 0;
```

Comment:
It seems like this is always zero. Can it be made a constant instead of an instance variable?